domains/zine.json

```diff
@@ -4,6 +4,6 @@
         "email": "zmoualhi@outlook.com"
     },
     "record": {
-        "URL": "https://muandane.github.io/portfolio/"
+        "CNAME": "muandane.github.io"
```

Comment:
CNAME returns a 404.

Reply:
How do you create a record for a GitHub pages url ?